core/templates/dev/head/player/exploration_player.html

```diff
@@ -95,7 +96,8 @@ <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
               src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
             <li ng-repeat="ContributorUsername in explorationContributorUsernames| limitTo: 2"
               tooltip="<[ContributorUsername]>" tooltip-placement="top">
-              <img class="oppia-contributors-circle" ng-src="/images/avatar/user_mint_48px.png" alt="">
+              <profile-link-image username="<[ContributorUsername]>" classes="oppia-contributors-circle" alt="">
```

Comment:
For now, I think, let's keep it simple and try to make it self-contained. We can add customizations later when they are really needed, but I think that doing it now is premature and makes the directive overly complicated.